### PR TITLE
🌱 replace inet.af/netaddr with net/netip

### PR DESCRIPTION
**Pull request by schrej**
**What this PR does / why we need it**:
Replaces `inet.af/netaddr` with `ip/netip`, which was added with Go 1.18.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -162,6 +161,4 @@ require (
 	github.com/google/gnostic v0.5.7-v3refs // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
 	github.com/pelletier/go-toml/v2 v2.0.1 // indirect
-	go4.org/intern v0.0.0-20211027215823-ae77deb06f29 // indirect
-	go4.org/unsafe/assume-no-moving-gc v0.0.0-20220617031537-928513b29760 // indirect
```
Comment (sbueringer) [inline go.mod:166]: That one sounds like fun :)

Comment (killianmuldoon) [reply go.mod:166]: lol

>If your Go package wants to declare that it plays unsafe games that only work if the Go runtime's garbage collector is not a moving collector, then add:
>
> `import _ "go4.org/unsafe/assume-no-moving-gc"`
>
> Then your program will explode if that's no longer the case. (Users can override the explosion with a scary sounding environment variable.)

**Review by killianmuldoon — APPROVED** [review]
/lgtm

File: go.mod (hunk 1 above)
Comment (killianmuldoon) [reply go.mod:166]: `
There is no API.
`
From the [godoc](https://pkg.go.dev/go4.org/unsafe/assume-no-moving-gc#section-readme)

Comment (sbueringer) [reply go.mod:166]: Probably good that we're not using it anymore

File: go.mod [hunk 2]
```diff
@@ -32,7 +32,6 @@ require (
 	golang.org/x/net v0.0.0-20220812174116-3211cb980234
 	golang.org/x/oauth2 v0.0.0-20220608161450-d0670ef3b1eb
 	google.golang.org/grpc v1.47.0
-	inet.af/netaddr v0.0.0-20220617031823-097006376321
```
Comment (sbueringer) [inline go.mod:35]: @schrej Doesn't matter at all. But are you also confused why we didn't have to bump `go 1.17` in the go.mod file to use `netip.ParseAddr`? :)

Comment (schrej) [reply go.mod:35]: Interesting. The compiler does heavily complain about generics if it doesn't say `1.18` in `go.mod`, but it doesn't seem to care about packages that aren't available in an earlier release.

It doesn't compile with 1.17, so technically the project needs to change to `1.18` in `go.mod`, or revert this change. Probably doesn't have any impact to consumers of APIs though, as its unlikely that someone imports the webhooks :smile:

Comment (schrej) [reply go.mod:35]: There is an issue for it: https://github.com/golang/go/issues/46136

Comment (sbueringer) [reply go.mod:35]: Thx for looking that up!

I think for CAPI main it's fine. We're probably days away from bumping to Go 1.19

**Pull request merged**